src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -141,6 +133,20 @@ export default {
 		};
 	},
 	computed: {
+		loanIds() {
+			console.log(_map(this.loans, 'id'));
+			return _map(this.loans, 'id');
+		},
```

Comment:
Please refactor to use optional chaining. We're slowing removing any use of lodash as we refactor or update code so please anytime you see something like this, switch to using plain js.